src/io/flutter/run/daemon/FlutterAppListener.java

```diff
@@ -105,9 +105,19 @@ public void onAppLog(@NotNull DaemonEvent.AppLog message) {
   @Override
   public void onAppProgressStarting(@NotNull DaemonEvent.AppProgress event) {
     progress.start(event.message);
+
     if (event.getType().startsWith("hot.")) {
+      // We clear the console view in order to help indicate that a reload is happening.
```

Comment:
Yeah, I'm really of two minds about this. It does to its job of making reload clearer - when it's happening, and when it's finished. But it also has the drawback of removing (potentially valuable) console history.

Preserving history w/ verbose sounds useful. A larger 'reloading' banner is a good idea but may not move the needle greatly.

I _think_ I'm leaning towards clearing the log on reload when verbose is not on. We can try that for a while and see if it's a net benefit.